src/main/resources/collaboratorPage.json

```diff
@@ -0,0 +1,40 @@
+{
+  "page": {
+    "title": "Team",
+    "subtitle": "Core Team",
+    "description": "The core team of our community is composed of visionary directors and dynamic leaders, who bring a wealth of experience and dedication to empowering women in technology. These esteemed professionals are committed to fostering an environment of growth, inclusivity and support, guiding our community toward achieving its mission and goals."
+  },
+  "contact": {
+    "title": "Contact us",
+    "links": [
+      {
+        "type": "email",
+        "link": "london@wcc.com"
+      },
+      {
+        "type": "slack",
+        "link": "http://shortlink_to_slack.com"
+      }
+    ]
+  },
+  "collaborators": [
+    {
+      "fullName": "Full name",
+      "position": "Senior Software Engineer",
+      "images": [
+        {
+          "path": "https://cloudprovider.com/image.png",
+          "alt": "There are two women talking during a productive mentoring session",
+          "type": "desktop"
+        }
+      ],
+      "network": [
+        {
+          "type": "youtube",
+          "link": "https://www.linkedin.com"
+        }
+      ],
+      "memberType": "director"
```

Comment:
Here you have to add the same collaborators we have in the current website. (No directors, leaders or evangelist)

Reply:
No, I don't think so. You can just add the current collaborators we have in our website.

Reply:
@dricazenck Ok. I updated the MemberType enum with WEBSITE_COLLABORATOR and TEAM_COLLABORATOR items but in the current way of enum implementation the items are displayed on the page with the underscore as well. I tried to change the enum to use the text value like this: WEBSITE_COLLABORATOR("WEBSITE COLLABORATOR") but that requires then other changes in the deserializer() function. We can discuss this later maybe.